test/chart.test.ts

```diff
@@ -142,7 +142,6 @@ test('synthesizeManifest() can be used to synthesize a specific chart', () => {
 });
 
 test('addDependency', () => {
-
```

Comment:
@shinebayar-g can you also remove these deleted lines? Think this was also caused by prettier formatting